tools/agent-ctl/src/client.rs

```diff
@@ -744,21 +768,24 @@ fn readline(prompt: &str) -> std::result::Result<String, String> {
 }
 
 fn agent_cmd_health_check(
-    cfg: &Config,
+    _cfg: &Config,
     _client: &AgentServiceClient,
     health: &HealthClient,
+    ctx: &Context,
```

Comment:
Since `cfg` is not used anymore, how about replacing it with `ctx`, let `ctx` as the first parameter like Golang.

Reply:
@liubin - agreed - fixed! :smile: